alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
```

Comment:
I tend to always use the "verbose" string iterpolation form in shell scripts since that one is less error prone on refactorings.